### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/alert.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#789] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/alert.html

主题可以提供 GFM 警告支持。 配置 演示 注 注释文字 重要 重要文字 提示 提示文字 注意 注意文字 警告 警告文字

https://theme-hope.vuejs.press/zh/guide/markdown/alert.html

**Comment by raysun0501** [comment]
请问容器标题如何自定义？
比如把“注”换成“摘要”

**Reply by Mister-Hope** [reply]
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/33315834/28a21dfa-3111-44d2-bde7-73ea34a8cd6e)

- https://theme-hope.vuejs.press/zh/guide/markdown/alert.html
- https://plugin-md-enhance.vuejs.press/zh/config.html#locales

**Reply by raysun0501** [reply]
感谢 Mr. Hope 的回复，看了locales，不过不知道咋配置，没太配置好...
最后是通过
::: note 摘要
xxx
:::
这种方式实现的，不知道和版主说的方式区别在什么地方？
是版主提供了2种不同的实现方式嘛😃
